CMakeLists.txt

```diff
@@ -360,6 +276,18 @@ install(FILES COPYING CHANGELOG.md
   DESTINATION "${CMAKE_INSTALL_DOCDIR}"
   COMPONENT runtime)
 
+# install third_party
+if (INSTALL_VENDORED_HEADERS)
+  install(DIRECTORY ${CMAKE_SOURCE_DIR}/third_party/rapidjson/include/
+    DESTINATION "${CMAKE_INSTALL_INCLUDEDIR}/valhalla/third_party")
+  install(DIRECTORY ${CMAKE_SOURCE_DIR}/third_party/date/include/
+    DESTINATION "${CMAKE_INSTALL_INCLUDEDIR}/valhalla/third_party")
+  if (WIN32)
+    install(DIRECTORY ${CMAKE_SOURCE_DIR}/third_party/dirent/include/
+      DESTINATION "${CMAKE_INSTALL_INCLUDEDIR}/valhalla/third_party")
+  endif()
+endif()
```

Comment:
good catch, but also good motivation to reorganize our public interface to move these to implementation and out of our headers